### ISSUE TITLE
Add bus channel feedbacks

### ISSUE THREAD

**Issue by nacalai** [bitfocus/companion-module-behringer-xair#44]
There seems to be missing feedbacks for bus channels. There are only for bus masters. Would love to have them added :)

**Comment by istnv** [comment]
Nearly every process/mute/send on the console is available. Which one do you think is missing?
Explain how to 'set' that using the X-Air Edit software, so I know what you are tying to find.

**Comment by nacalai** [comment]
@istnv It might just me being blind, but I did search for it twice and couldn't find it. To elaborate:

Select a bus in xair, let's say Bus 1
![image](https://user-images.githubusercontent.com/4357694/131700242-3dd29458-8b78-4b8b-b400-9de60c1aadf2.png)
Then select a bus channel, let's say channel 1
![image](https://user-images.githubusercontent.com/4357694/131700288-995516a2-d650-4d7f-bcd0-9352f6960789.png)

I can find status feedbacks for LR channels, FX channels, DCA channels and all masters, but not these bus channels. They DO exists as variables. For this instance, to get the db, it's $(xair:s_ch1_b1_d). But seemingly it does not exist as a feedback.

**Comment by nacalai** [comment]
Right. It just hit me. I can of course add companions internal feedback called "Check variable value" in which I can get all the variables from. Dumb me. Alright. BUT, it would be nice to have it as feedbacks anyway :) Seeing as you can find feedbacks for everything else.

**Comment by istnv** [comment]
There are *feedbacks* for the on/off actions like Mutes and Solos: Change color to Red if Muted.
There are *variables*  for values that have more than 2 (on/off) values: Ch1 Fader level, Ch1 Send to Bus1 Level.
If you were looking for a Bus Mute per channel, the XAir models do not have one. (M/X 32 series does)

**Comment by nacalai** [comment]
Aha..that explains it. I did not realize that there is in fact not a bus channel specific mute on these models. That went totally over my head. I'll just use variables then, all good. Thanks for clearing it up.
